### PR TITLE
Feature/image optimization png jpg

### PR DESCRIPTION
**Pull request by CrazyOptimist**
Optimize images for Add NFT using pngquant for png and mozjpeg for jpg

Added `pngquant` and `mozjpeg` static binaries to `static/bin`

After user selects a picture it starts processing 6 versions of different quality: 40% quality, 50% and so on until 90%

Optimization slider allows user to see steps with size in MB

image

(shadows in app looks better than on screenshot)

### CODE REVIEW

**Review by sanjanadhikari1 — APPROVED** [review]
It looks good for me

**Review by ngvtuan — COMMENTED** [review]
upload issue

This issue happens if we changed the quality of the image

**Comment by CrazyOptimist** [comment]
@ngvtuan could you please tell where you see this json and how to reproduce?

File: src/common/components/StepsCircle/StepsCircle.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    <defs>
+      <linearGradient
+        id='paint0_linear'
+        x1='-1'
+        y1='0'
+        x2='-1'
+        y2='64'
+        gradientUnits='userSpaceOnUse'
+      >
+        <stop stop-color='#3F9AF7' />
```
[the remark this answers is not in the document]
Comment (CrazyOptimist) [reply src/common/components/StepsCircle/StepsCircle.tsx:33]: fixed

**Comment by ngvtuan** [comment]
> @ngvtuan could you please tell where you see this json and how to reproduce?

@CrazyOptimist Here is the steps: 
- Upload an image step-1
- Go to the `optimize` select quality is 35% step-2
- Save data  step-3

Then you will see the issue.

**Comment by CrazyOptimist** [comment]
Oh, walletnode is not ready to use, so it can't be a blocker in this PR, @ngvtuan

**Review by ngvtuan — APPROVED** [review]

**Review by sanjanadhikari1 — APPROVED** [review]

**Pull request merged**